textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/utils/TestUtils.java

```diff
@@ -32,6 +38,40 @@ public static boolean contains(List<ITuple> sampleTuples, ITuple actualTuple, Li
         return contains;
     }
     
+    public static boolean containsAllResults(List<ITuple> expectedResults, List<ITuple> exactResults) {
+        if(expectedResults.size() != exactResults.size())
+        	return false;
+        if(!(expectedResults.containsAll(exactResults)) || !(exactResults.containsAll(expectedResults)))
+        	return false;
+        
+        return true;
+    }
+    
+    public static boolean checkResults(List<ITuple> results, String queryString, Analyzer queryAnalyzer, String searchField) throws ParseException {
+        
+    	
+    	boolean contains = false;
+        QueryParser parser = null;
+        parser = new QueryParser(searchField, queryAnalyzer);
+        Query query = parser.parse(queryString);
+        String searchString = query.toString(searchField);
+        System.out.println(query.toString(searchField));
+        String[] listOfQueryWords = searchString.split(" ");

```

Comment:
Here's how I did:

According to their current implementation let's say the query is  "short,tall"  they are returning all the documents which contains either short or tall in the specified field.  When we use StandardAnalyzer(which team 4 is currently using) for queryString the above query after parsing it'll be "short tall". 

Query query = parser.parse(queryString);

query.toString will return "short tall"

Here we're not searching for the query, If we are searching for the query just the query is enough no need to split it, lucene searcher will take care of it. I just want the individual words "short" and "tall" to check if either of them is present in the returned results or not that's why used split on the query. This method covers all the test cases for their current implementation. I'll discuss more about this in tomorrow's class.